acceptance/support-skeletons/synced_folder_smb/Vagrantfile

```
@@ -0,0 +1,8 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "box"
+
+  config.vm.synced_folder ".", "/vagrant-smb", type: "smb", smb_username: ENV["SMB_USERNAME"], smb_password: ENV["SMB_PASSWORD"]
```

I'd suggest naming these environment variables with a `VAGRANT_` prefix so it's clearer how they're being used.

```suggestion
  config.vm.synced_folder ".", "/vagrant-smb", type: "smb", smb_username: ENV["VAGRANT_SMB_USERNAME"], smb_password: ENV["VAGARANT_SMB_PASSWORD"]
```